apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/common/FissTransformerV2.java

```diff
@@ -0,0 +1,52 @@
+package gov.cms.bfd.server.war.r4.providers.pac.common;
+
+import gov.cms.bfd.model.rda.RdaFissClaim;
+import gov.cms.bfd.model.rda.RdaFissPayer;
+import java.util.Map;
+import java.util.Optional;
+import org.hl7.fhir.r4.model.Enumerations;
+import org.hl7.fhir.r4.model.Patient;
+import org.hl7.fhir.r4.model.Resource;
+
+public class FissTransformerV2 {
+
+  private static final Map<String, Enumerations.AdministrativeGender> GENDER_MAP =
+      Map.of(
+          "m", Enumerations.AdministrativeGender.MALE,
+          "f", Enumerations.AdministrativeGender.FEMALE,
+          "u", Enumerations.AdministrativeGender.UNKNOWN);
+
+  private FissTransformerV2() {}
+
+  public static Resource getContainedPatient(RdaFissClaim claimGroup) {
+    Optional<RdaFissPayer> optional =
```

Comment:
This should probably have a more descriptive name (per below, probably benePayerOptional)